packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -164,14 +165,20 @@ WebView::WebView(flutter::PluginRegistrar* registrar, int viewId,
       height_(height),
       working_surface_(nullptr),
       candidate_surface_(nullptr),
-      rendered_surface_(nullptr),
       is_mouse_lbutton_down_(false),
+      rendered_surface_(nullptr),
       has_navigation_delegate_(false),
       has_progress_tracking_(false),
       context_(nullptr),
       texture_variant_(nullptr),
       platform_window_(platform_window) {
-  tbm_pool_ = std::make_unique<BufferPool>(width, height);
+  use_sw_backend_ = NeedsSwBackend();
+  if (use_sw_backend_) {
+    tbm_pool_ = new SingleBufferPool(width, height);
+  } else {
+    tbm_pool_ = new BufferPool(width, height, BUFFER_POOL_SIZE);
+  }
```

Comment:
Why don't you use [unique_ptr](https://en.cppreference.com/w/cpp/memory/unique_ptr) anymore?

``` c++
std::cout << "\n" "2) Runtime polymorphism demo\n";
{
    // Create a derived resource and point to it via base type
    std::unique_ptr<B> p = std::make_unique<D>();

    // Dynamic dispatch works as expected
    p->bar();
}
```

Reply:
Well, there's no other reason. When I first did it, an error occurred and I just thought it wouldn't work.
I'll try it again.

Reply:
done!